dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/configurator/parser/ConfigParser.java

```diff
@@ -137,8 +137,6 @@
 
     private static String toParameterString(ConfigItem item) {
         StringBuilder sb = new StringBuilder();
-        sb.append("category=");
-        sb.append(DYNAMIC_CONFIGURATORS_CATEGORY);
```

Comment:
I think we should keep this two lines and remove `urlBuilder.append("&category=").append(DYNAMIC_CONFIGURATORS_CATEGORY);` 
in the caller mehod.

Reply:
Yes, you are right. it's my fault that I didn't look carefully. method appItemToUrls also calls the method toParameterString, and it is filled with the "appdynamicconfigurators" parameter.Indeed, only the serviceItemToUrls method has the "category" parameter repeatedly added.

Reply:
Thanks. All look fine now.

Reply:
Hi, do you have time to fix this issue so we can merge this patch soon？

Reply:
I have modified it, but there are multiple commit logs under this PR, do I need to re-issue a new PR request to merge?